docs/maya/simple_setup.md

```diff
@@ -63,6 +63,50 @@ The order of painting is important because after every stroke a normalisation of
 
 With this basic paint setup the AdnSkin deformer will already show plausible results, expected of the skin to the target mesh. However, the possible parameters and tweaks to display high fidelity dynamics can be seen in the documentation for [AdnSkin](skin).
 
+## AdnFat
+
+To create a basic scenario using the AdnFat deformer, start with a scene with the following elements:
+
+  - One base mesh to which the fat layer will be attached to. This could be for example the fascia.
+  - A fat tissue mesh without animation or deformation.
+
+The AdnFat deformer will get applied to the second selected mesh which will become the simulated mesh (the fat tissue).
+
+<figure>
+  <img src="images/setup_fat_0.png"> 
+  <figcaption><b>Figure 7</b>: Basic setup for fat simulations.</figcaption>
+</figure>
+
+> [!NOTE]
+> - The base mesh and the fat mesh must have the same vertex count and triangulation.
+> - The base mesh must be provided, otherwise the Fat solver will abort the simulation.
+
+### Create Deformer
+
+To create the AdnFat deformer first select the base mesh and then the fat tissue mesh. Then press the ![AdnFat](images/adn_fat.png){style="width:4%"} shelf button or go to AdonisFX Menu > *Fat*.
+
+To create the AdnFat deformer with some initial customisation, double-click the shelf button or press the option box in the menu item. This will display a pop-up window that will allow to do some initial customisation, as well as creating the deformer with a custom name. Once all data has been provided press the *Create* button and the deformer will get created.
+
+<figure>
+  <img src="images/setup_fat_1.png"> 
+  <figcaption><b>Figure 8</b>: AdnFat deformer creation scenario.</figcaption>
+</figure>
+
+After basic configuration, to alter the dynamics of the fat layer (adding or reducing the jiggle effect) it is advisable to tweak the main attributes like; *Iterations*, *Substeps*, *Global Damping Multiplier* or the per-constraints stiffness values in the *Override Constraint stiffness* section.
+
+### Paint Weights
+
+With the default paint setup provided when creating a new AdnFat, the simulation should already create plausible results. However, below we walk you through the 2 main maps that can be altered to modify the behavior of the fat simulation.
+
+In the case of the AdnFat deformer use the Maya Paint tool to setup and paint its paintable weight attributes. The most important paintable maps are *Shape Preservation* and *Volume Shape Preservation* both with a flooded value of 1.0 by default.
```

Comment:
I would personally reiterate on the use of the maya paintable context in a Note, this will make it clearer.

I think we don't do so in AdnSimshape. However, it happened to Francesco that he intuitively tried to use our tool. We might encounter ourselves receiving tickets from users saying that AdonisFX paint tool does not work for fat.

Reply:
Added it to the beginning of the Paint Weights section both in Simshape and Fat

Reply:
Also to SkinMerge